core/modules/ccontrol/queryMsg.cc

```diff
@@ -43,19 +43,23 @@ int queryMsgGetCount(int session) {
 }
 
 // Python call: msg, chunkId, code, timestamp = queryMsgGetMsg(session, idx)
-std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, time_t* timestamp) {
+std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, std::string* severity, time_t* timestamp) {
```

Comment:
See below for the arguments.  For the return value, a reference to string is somewhat dangerous and only appropriate if you know that the underlying string has a lifetime as long or longer than any user of the result (e.g. because it is a constant).